problems/numberOfDigits.js

```diff
@@ -7,8 +7,27 @@
 
 */
 
-function numberOfDigits() {
-
+function numberOfDigits(n) {
+    let num = n
+    return Math.floor(num.toString().length)
```

Comment:
No need for Math.floor here. A string will always be an integer length and never a decimal

Reply:
Noted.